include/deal.II/fe/mapping.h

```diff
@@ -962,16 +962,15 @@ class Mapping : public Subscriptor
    *   this object also represents with respect to which cell the transformation
    *   should be applied to.
    * @param[out] output An array (or part of an array) into which the transformed
-   *   objects should be placed.
+   *   objects should be placed. (Note that the array view is @p const, but the
+   *   tensors it points to are not.)
```

Comment:
This comment makes me wonder whether the output array view should really be constant in terms of readability. Sure, a const view to non-const data allows to modify the data pointed to, but I find it much more explicit to have the output arguments marked as non-const (in case someone does not bother to read the documentation). I must say that while I understand the C++ rules of const declarations regarding pointers, I do not find them particularly intuitive, thus my preference for either `double const` or `no const`.